clang/lib/AST/ASTImporter.cpp

```diff
@@ -3636,6 +3636,10 @@ class IsTypeDeclaredInsideVisitor
     return {};
   }
 
+  std::optional<bool> VisitUnaryTransformType(const UnaryTransformType *T) {
+    return CheckType(T->getBaseType());
```

Comment:
Strictly speaking, you only need to visit the base type if the `UnaryTransformType` is not sugared, which currently is the same as being dependent.

Otherwise previous CheckType should have already stepped through it.